src/main/kotlin/blackjack/entity/Player.kt

```diff
@@ -1,18 +1,20 @@
 package blackjack.entity
 
-// todo: limit 이 static 한 변수일 때, inferface에서 선언하는 방법
+import blackjack.ui.GetResult
+import blackjack.ui.Input
+
 class Player(override val name: String, override val wallet: Wallet, override val limit: Int = 21) : Person {
-    override fun draw(): Wallet {
+    override fun draw(wallet: Wallet): Wallet {
         val cards = wallet.getMutableCardList()
         cards.add(CardDrawer.drawSingleCard())
         return Wallet(cards)
     }
 
-    fun getWalletSum(): Int {
-        return wallet.sumUp
-    }
-
-    fun getWalletCards(): List<Card>{
-        return wallet.cards
+    override fun chooseDrawing(wallet: Wallet): Wallet {
+        if (!wallet.isAbleToDraw(limit)) return wallet
+        if (Input.additionalCard(name) == "n") return wallet
+        val newPlayer = Player(name, draw(wallet))
+        GetResult.printPlayerStatus(newPlayer)
+        return chooseDrawing(wallet)
```

Comment:
```suggestion
        if (!wallet.isAbleToDraw(limit)) return wallet
        if (Input.additionalCard(name) == "n") return wallet
        val newPlayer = Player(name, draw(wallet))
        GetResult.printPlayerStatus(newPlayer)
        return chooseDrawing(wallet)
```

view의 책임이 섞여 있는 것 같아요 :) 
mvc 패턴에 맞도록 적절하게 분리해주는건 어떨까요??

Reply:
이 함수도 아래와 같이 view에 의존성을 가지는 Blackjack 객체로 이동시켰습니다!

Reply:
다만 이렇게 하면 객체지향적이지 않은 코드가 되는 것 같아요. 원래 의도는 chooseDrawing을 dealer도 player 도 가지고 있고, 구현만 다르니 Person이라는 interface에 넣으려고 했어서 이 부분을 조금 고민해봐야할 것 같아요.